sdk/data/aztables/client.go

```diff
@@ -23,7 +23,6 @@ type Client struct {
 	service *ServiceClient
 	cred    *SharedKeyCredential
 	name    string
-	con     *generated.Connection
```

Comment:
Removing this would be a breaking change.  If it wasn't usable as it stood before, then its ok to remove.  Otherwise we should deprecate it instead.